### PR TITLE
OCPBUGS-31759: Update i18n docs on how to update Phrase Project Template

### PR DESCRIPTION
**Pull request by cyril-ui-developer**

### CODE REVIEW

File: INTERNATIONALIZATION.md [hunk 1]
```diff
@@ -109,6 +109,7 @@ To add support for a new language to OpenShift:
 1. Look up the [ISO 639-1 code](https://www.loc.gov/standards/iso639-2/php/code_list.php) for the new language.
 2. Add the new language code to `./frontend/i18n-scripts/languages.sh`
 3. Update the language switcher component (`./frontend/public/components/modals/language-preferences-modal.tsx`) to support the new language if translations are available.
+4. Update the OCP UI Project Template to include the new laguage(s) in the Phrase TMS portal. Refer to the [Phrase Project Templates ](https://support.phrase.com/hc/en-us/articles/5709647439772-Project-Templates-TMS) on how to update Project Template.  Please note that you must have Phrase project owner permissions to perform this task.
```
Comment (jhadvig) [inline INTERNATIONALIZATION.md:112]:
```suggestion
4. Update the OCP UI Project Template to include the new laguage(s) in the Phrase TMS portal. Refer to the [Phrase Project Templates ](https://support.phrase.com/hc/en-us/articles/5709647439772-Project-Templates-TMS) on how to update Project Template. Please note that you must have Phrase project owner permissions to perform this task.
```

File: INTERNATIONALIZATION.md [hunk 2]
```diff
@@ -119,3 +120,6 @@ For more information, please review the [README](./frontend/i18n-scripts/README.
...
+
+#### Supported Languages representation
+The languages that are supported are represented using the ISO 639-1 Code, as defined in `frontend/packages/console-app/src/components/user-preferences/language/const.ts`. An exception is made for Chinese, where the region code is included to distinguish the supported traditional Chinese from other variants. However, the locale directories for Chinese still follow the ISO 639-1 Code format.
```
Comment (jhadvig) [inline INTERNATIONALIZATION.md:125]:
```suggestion
Supported languages are represented using the ISO 639-1 Code, as defined in `frontend/packages/console-app/src/components/user-preferences/language/const.ts`. An exception is made for Chinese, where the region code is included to distinguish the supported traditional Chinese from other variants. However, the locale directories for Chinese still follow the ISO 639-1 Code format.
```

**Review by jhadvig — CHANGES_REQUESTED** [review]
Thanks @cyril-ui-developer for the PR.
Adding couple of comments.
@opayne1 PTAL as well.

**Comment by cyril-ui-developer** [comment]
@jhadvig Updated, PTAL

File: INTERNATIONALIZATION.md [hunk 3]
```diff
@@ -119,3 +120,6 @@ For more information, please review the [README](./frontend/i18n-scripts/README.
...
+
+#### Supported Languages representation
+Supported languages are represented using the ISO 639-1 Code, as defined in `frontend/packages/console-app/src/components/user-preferences/language/const.ts`. An exception is made for Chinese, where the region code is included to distinguish the supported traditional Chinese from other variants. However, the locale directories for Chinese still follow the ISO 639-1 Code format.
```
[the remark this answers is not in the document]
Comment (opayne1) [reply INTERNATIONALIZATION.md:125]: It should probably be "There is an exception for Chinese" or "with the exception of Chinese". What do you think of this re-write?

```suggestion
Supported languages are represented using the ISO 639-1 Code defined in `frontend/packages/console-app/src/components/user-preferences/language/const.ts` with the exception of Chinese. While locale directories for Chinese still follow the ISO 639-1 Code format, the region code is also included to distinguish the supported traditional Chinese from other variants.
```

File: INTERNATIONALIZATION.md [hunk 4]
```diff
@@ -109,6 +109,7 @@ To add support for a new language to OpenShift:
 1. Look up the [ISO 639-1 code](https://www.loc.gov/standards/iso639-2/php/code_list.php) for the new language.
 2. Add the new language code to `./frontend/i18n-scripts/languages.sh`
 3. Update the language switcher component (`./frontend/public/components/modals/language-preferences-modal.tsx`) to support the new language if translations are available.
+4. Update the OCP UI Project Template to include the new laguage(s) in the Phrase TMS portal. Refer to the [Phrase Project Templates ](https://support.phrase.com/hc/en-us/articles/5709647439772-Project-Templates-TMS) on how to update Project Template. Please note that you must have Phrase project owner permissions to perform this task.
```
Comment (opayne1) [inline INTERNATIONALIZATION.md:112]:
```suggestion
4. Update the OCP UI Project Template to include the new laguage(s) in the Phrase TMS portal. Refer to the [Phrase Project Templates ](https://support.phrase.com/hc/en-us/articles/5709647439772-Project-Templates-TMS) on how to update the Project Template. You must have Phrase project owner permissions to perform this task.
```

File: INTERNATIONALIZATION.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (cyril-ui-developer) [reply INTERNATIONALIZATION.md:125]: @opayne1 @jhadvig I think the ""with the exception of Chinese"" looks and sound better.

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve
/retest

**Comment by jhadvig** [comment]
hmmm looks like CI is down

**Pull request merged**